ci-operator/step-registry/rosa/terraform/aws/e2e/test/rosa-terraform-aws-e2e-test-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+
+# Configure aws
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+  export AWS_DEFAULT_REGION="${CLOUD_PROVIDER_REGION}"
+else
+  echo "Did not find compatible cloud provider cluster_profile"
+  exit 1
+fi
+
+
+mkdir -p ~/.aws
+cp ${AWSCRED} ~/.aws/credentials
+
+TEST_OFFLINE_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+export TF_VAR_token=${TEST_OFFLINE_TOKEN}
+
+cp -r /root/terraform-provider-ocm ~/
+cd ~/terraform-provider-ocm
+
+export GOCACHE="/tmp/cache"
+export GOMODCACHE="/tmp/cache"
+export GOPROXY=https://proxy.golang.org
+go mod download
+go mod tidy
+go mod vendor
```

Comment:
Maybe duplicate with https://github.com/terraform-redhat/terraform-provider-ocm/blob/main/build/ci-tf-e2e.Dockerfile#L28, and why not build the GO variables into the docker file?